src/include/crowbar/complex.rb

```diff
@@ -115,7 +115,7 @@ def initialize_crowbar_complex(include_target)
               "It is also possible to use custom paths. Some examples of how the URL could look like:\n" +
               "</p><p>\n" +
               "<ul>\n" +
-              "<li><i>http://smt.example.com/repo/SUSE/Products/SLE-HA/12-SP2/x86_64/product</i> for SMT server\n" +
+              "<li><i>http://smt.example.com/repo/SUSE/Products/SLE-HA/12-SP4/x86_64/product</i> for SMT server\n" +
```

Comment:
Hm, this will break all translations... :worried: 

Ideally URLs or any variable texts should not be included in the translated message, there should be just some place holders which get replaced by the real value *after* translating the text.

```ruby
_("<p>Here you can edit ... " \
  ...
  "<li><i>%{ha_url</i> for SMT server</li>\n" \ 
  "<li><i>%{suma_url}</i> for SUSE Manager Server.</li>\n" \
  "...") % {
    ha_url: "http://smt.example.com/repo/SUSE/Products/SLE-HA/12-SP2/x86_64/product",
    suma_url: "http://manager.example.com/ks/dist/child/suse-openstack-cloud-7-pool-x86_64/sles12-sp2-pool-x86_64/" 
  }
```

*(Please double check, hopefully I didn't do any mistake there...)*

This change will of course also break the translations, but the next time we need to update the URLs it will not break anything™.

### Some Notes

- You can use `\` for concatenating the string parts together, that's more efficient way for joining string literals in Ruby.
- The `</li>` closing tags were missing. It seems that the parser ignores this issue (autocloses them), but it should be fixed anyway.

Reply:
I think we're actually not translating this at all ...

Reply:
The texts actually *are* translated, see https://github.com/yast/yast-translations/tree/master/po/crowbar.

If you do not want to translate them and you want to save some translation efforts then ping Stanislav Brabec, he can add it into the ignore list...